### PR TITLE
2023 Scenario Geometry

### PR DESCRIPTION
**Pull request by ianna**
- Forward port PhaseII beampipe
- Forward port materials and merge with existing description
- Forward port Muon RPC and GEM geometry description
- Add Extended2023 label for geometry configuration for cmsDriver
- Update scenario

Step 1 runs with:

```
runTheMatrix.py -l 2.0 --command "--geometry Extended2023"
```

NOTE: an appropriate GT may be needed

### CODE REVIEW

**Comment by ianna** [comment]
+1

it does not affect any existing workflows and may be considered as a bug-fix.

@slava77 and @dildick - FYI. Muon group may want to check if all xml files for PhaseII are up-to-date.

File: Configuration/Geometry/python/GeometryExtended2023Reco_cff.py [hunk 1]
```diff
@@ -15,13 +15,12 @@
 #Muon
 from Geometry.MuonNumbering.muonNumberingInitialization_cfi import *
 from RecoMuon.DetLayers.muonDetLayerGeometry_cfi import *
-from Geometry.GEMGeometry.gemGeometry_cfi import *
```
Comment (dildick) [inline Configuration/Geometry/python/GeometryExtended2023Reco_cff.py]: Hi Yana, 

This file was indeed deleted. Could you instead add the following lines:
<PRE>
from Geometry.GEMGeometryBuilder.gemGeometry_cfi import *
from Geometry.GEMGeometryBuilder.me0Geometry_cfi import *
</PRE>

Comment (ianna) [inline Configuration/Geometry/python/GeometryExtended2023Reco_cff.py]: @dildick - thanks! I'll add them. However, the 2023 scenario does not have ME0 in. Should it be added as well?

Comment (dildick) [inline Configuration/Geometry/python/GeometryExtended2023Reco_cff.py]: Just to clarify, is Extended2023 supposed to have ME0? If not, it shouldn't be added.

**Comment by ianna** [comment]
+1

**Comment by davidlange6** [comment]
@ianna- which of the 62x scenarios does this "2023" scenario correspond to?

**Comment by ianna** [comment]
@davidlange6 - eventually, Extended2023, but neither Tracker nor Hcal is ready, so it's only PhaseII Muons and beampipe with other current subdetectors

**Comment by davidlange6** [comment]
@ianna- that sounds good to me - however from what scenario in 62xSLHC were the muons and beam pipe taken from?

**Comment by ianna** [comment]
@davidlange6 - a HEAD of Extended2023 on the day (ME0 is not there, there is no space for it with current Hcal), though all 2023 scenarios in 62x_SLHC have the same PhaseII beam pipe, RPCs, CSCs and GEM (GE1/1,2/1)

**Comment by davidlange6** [comment]
closing in 74x - there is the same PR open in 75x (now the development release)

**Comment by ianna** [comment]
@davidlange6 - please, reopen and accept this PR as a bug fix. It removes references to non-existing files.

**Pull request closed without merging**